services/web/server/src/simcore_service_webserver/computation_subscribe.py

```diff
@@ -2,56 +2,86 @@
 import json
 import logging
 from functools import wraps
+from pprint import pprint
+from typing import Any, Callable, Coroutine, Dict
 
 import aio_pika
 from aiohttp import web
+
 from servicelib.application_keys import APP_CONFIG_KEY
 from simcore_sdk.config.rabbit import eval_broker
 
+from .computation_api import get_task_output
 from .computation_config import (APP_CLIENT_RABBIT_DECORATED_HANDLERS_KEY,
                                  CONFIG_SECTION_NAME)
+from .projects import projects_api
 from .resource_manager.websocket_manager import managed_resource
 from .socketio.config import get_socket_server
 
 log = logging.getLogger(__file__)
 
 
-def rabbit_handler(app: web.Application):
+def rabbit_handler(app: web.Application) -> Callable:
     """this decorator allows passing additional paramters to python-socketio compatible handlers.
     I.e. aiopika handler expect functions of type `async def function(message)`
     This allows to create a function of type `async def function(message, app: web.Application)
     """
-    def decorator(func):
+    def decorator(func) -> Coroutine:
         @wraps(func)
-        async def wrapped(*args, **kwargs):
+        async def wrapped(*args, **kwargs) -> Coroutine:
             return await func(*args, **kwargs, app=app)
         return wrapped
     return decorator
 
 
-async def on_message(message: aio_pika.IncomingMessage, app: web.Application):
+async def post_messages(app: web.Application, user_id: str, messages: Dict[str, Any]) -> None:
```

Comment:
MINOR: Improve clarity. I find confusing the name ``post_message`` in a module that has code for rabbit ( a message broker) and websockets (that exchange messages with the front-end).

Reply:
ok I separated the modules. it should be clearer and PRed to Odeis PR.